src/main/java/com/kobylynskyi/graphql/codegen/model/MappingContext.java

```diff
@@ -259,6 +259,11 @@ public Boolean getUseOptionalForNullableReturnTypes() {
         return config.getUseOptionalForNullableReturnTypes();
     }
 
+    @Override
```

Comment:
we should also add it to sbt. This is easy to write

Reply:
Can you give me a hint on where/how that would be done? A bit out of my depth, but happy to give it a try.

Reply:
I tried adding it: [Commit](https://github.com/kobylynskyi/graphql-java-codegen/pull/1450/commits/da664558f25925bdbdb65262912bd4cb259d073f)
Let me know if this covers all changes needed for SBT.